src/views/Login.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { handleTextInput, handleSignIn } from "../redux/actions/loginActions";
+import { isEmpty } from "../utils/helperFunctions";
+import Validator from "../utils/validator";
+import TextInput from "../components/common/Inputs/TextInput";
+import FormButton from "../components/common/Buttons/FormButton";
+import BasicButton from "../components/common/Buttons/BasicButton";
+import SocialButton from "../components/common/Buttons/SocialButton";
+
+export class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      errors: {}
+    };
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleNavigation = this.handleNavigation.bind(this);
+  }
+
+  handleOnChange(e) {
```

Comment:
Use arrow functions and hence no need for binding

Reply:
Our .eslint is complaining about arrow functions. So, can I adjust the configuration first before implementing arrow functions?